scripts/modules/cli.py

```diff
@@ -379,6 +384,12 @@ def init_start_parser(self, parser, services, argv=None):
             default="false"
         )
 
+        parser.add_argument(
+            '--output-format',
+            help='Select the output format for the docker-compose.yml file. [yaml, json]',
+            default="yaml"
```

Comment:
Can we defer changing the default format till later? Having the additional option is great, but changing behaviour might break peoples' usage.